src/Arcus.Scripting.Tests.Integration/Connect-AzAccountFromConfig.ps1

```diff
@@ -2,7 +2,24 @@ param(
     [pscustomobject]$config
 )
 
- $clientSecret = ConvertTo-SecureString $config.Arcus.ServicePrincipal.ClientSecret -AsPlainText -Force
- $pscredential = New-Object -TypeName System.Management.Automation.PSCredential($config.Arcus.ServicePrincipal.ClientId, $clientSecret)
- Disable-AzContextAutosave -Scope Process
- Connect-AzAccount -Credential $pscredential -TenantId $config.Arcus.TenantId -ServicePrincipal
\ No newline at end of file
+$clientSecret = ConvertTo-SecureString $config.Arcus.ServicePrincipal.ClientSecret -AsPlainText -Force
+$pscredential = New-Object -TypeName System.Management.Automation.PSCredential($config.Arcus.ServicePrincipal.ClientId, $clientSecret)
+Disable-AzContextAutosave -Scope Process
+Connect-AzAccount -Credential $pscredential -TenantId $config.Arcus.TenantId -ServicePrincipal
+
+$tenantid = $config.Arcus.TenantId
+$body =  @{
+    Grant_Type    = "client_credentials"
+    Scope         = "https://graph.microsoft.com/.default"
+    Client_Id     = $config.Arcus.ServicePrincipal.ClientId
+    Client_Secret = $config.Arcus.ServicePrincipal.ClientSecret
+}
+
+$connection = Invoke-RestMethod `
+    -Uri https://login.microsoftonline.com/$tenantid/oauth2/v2.0/token `
+    -Method POST `
+    -Body $body
+ 
+$token = $connection.access_token
+
+Connect-MgGraph -AccessToken $token 
```

Comment:
Oh, cool, this helps us to connect to the Azure Graph API, right? Would it be a good thing if we split this in a separate file so we don't burden each integration test with this authentication process? If it could/should only be used in the Active Directory tests, we may want to separate it for now, right?

Reply:
Fixed in [`25cb335` (#336)](https://github.com/arcus-azure/arcus.scripting/pull/336/commits/25cb33526a99a3bcd008ff9f3475a80b402fbbbb)